content/en/docs/release-oversight/testing-in-production.md

```diff
@@ -0,0 +1,57 @@
+
+From there Quality Engineering will run additional tests.  Barring any new surprises Test Platform will be notified that a release is ready.
+
+It's worth noting that we may create a dedicated channel for managing these releases through the [Update Service](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html) in the upcoming months.  At the present we classify the sprint payload as an *fc* build which notifies Quality Engineering that additional testing is needed.
+
+### The environment
+We've selected one of our build farm clusters to run the output of each sprint.  This is a critical piece of infrastructure that all development teams are motivated to keep working.  In this document we'll refer to this cluster as the *sacrificial cluster*.
+
+### Investigating problems
+The TRT has admin access to the sacrificial cluster for debugging purposes.  As needed this access can be temporarily granted by Test Platform to any other development team that needs it.  Temporary configuration changes are allowed for experiments but anything needed to persist will require failing the sprint.
+
+### Hotfixes
+Hotfixes are possible but not desired.  Changes for the sole purpose of gathering more data are preferred over changes in functionality.  Before we resort to patching a problem we'll default to [reverting the change](/docs/release-oversight/improving-ci-signal/).  Changes like this are best done as part of the subsequent sprint release.
+
+### When things go wrong the sacrificial cluster is removed from rotation
+The Test Platform team maintains the infrastructure that powers our CI builds.  When there are major outages they already have an incident management process in place.  What's different now is that the TRT can detect subtle degradations as a result of sprint upgrades.  In most of these cases the TRT contacts Test Platform via their helpdesk bot to request the sacrificial cluster be removed from rotation.  Behind the scenes a Jira is created for tracking.
+
+This Jira is the rallying point for incident management.  Any changes made to the system are recorded in a root cause analysis document.  For example, rather than applying an actual hotfix (implying and entirely new upgrade) the TRT may decide to allow the cluster to enter a form of "rescue mode".  By this we mean disabling the cluster version operator and manually deploying isolated components.  Bugs that ship as part of our operating system content will not lend themselves to this sort of patching and will be handled on a case by case basis (See the [Reasonable effort](/docs/release-oversight/testing-in-production/#reasonable-effort-will-be-made-to-bring-the-sacrificial-cluster-back-into-rotation) section for more explanation).
```

Comment:
```suggestion
This Jira is the rallying point for incident management.  Any changes made to the system are recorded in a root cause analysis document.  For example, rather than applying an actual hotfix (implying an entirely new upgrade) the TRT may decide to allow the cluster to enter a form of "rescue mode".  By this we mean disabling the cluster version operator and manually deploying isolated components.  Bugs that ship as part of our operating system content will not lend themselves to this sort of patching and will be handled on a case by case basis (See the [Reasonable effort](/docs/release-oversight/testing-in-production/#reasonable-effort-will-be-made-to-bring-the-sacrificial-cluster-back-into-rotation) section for more explanation).
```